app-admin/passwordsafe/passwordsafe-1.06_beta-r1.ebuild

```diff
@@ -80,8 +80,8 @@ src_install() {
 	# The upstream Makefile builds this .zip file from html source material for
 	# use by the package's internal help system. Must prevent
 	# Portage from applying additional compression.
-	docompress -x /usr/share/doc/${PN}/help
-	insinto /usr/share/doc/${PN}/help
+	docompress -x /usr/share/doc/pwsafe/help
```

Comment:
My bad for not noticing this before but this location is forbidden. Only `/usr/share/doc/${PF}` is allowed.

Reply:
(and you're not supposed to let programs access that, it's purely user-oriented)